changelog.md

```diff
@@ -2,6 +2,10 @@
 
 //  Add your own contribution below
 
+### 0.6.1
+
+* upgrades to flow 0.35.0 and fixes associated type errors in covariant/invariant interfaces
```

Comment:
The JS community is so aggressive with releases, I hadn't even released the package yet. Maybe I need to get into this auto releasing package business for the JS version